setup.py

```diff
@@ -25,10 +14,13 @@ def get_version():
     include_package_data=True,
     zip_safe=False,
     platforms='any',
+    # Manage version number from git at dev-time or pkg_resources at run-time.
+    setup_requires=['setuptools-git-version'],
+    version_format='{tag}',
```

Comment:
Thanks for this pull request, @tcorbettclark! The only thing I don't like about it is the introduction of this install-time dependency on `setuptools-git-version`. It actually broke for me because of a dependency conflict with setuptools.

Suppose the latest version released to PyPI is 1.2.3. Then:

- On `master`, our WIP, version.py should report `1.2.4.dev0` to indicate a non-official version
- When ready, bump version.py to `1.2.4` (stable), tag it, release to PyPI, and change the version.py on `master` to `1.2.5.dev0`

This will also play well with `pip`'s rules for upgrading/downgrading.

I'll change this back to manual version tagging, then pull this PR. Thanks for the work!